packages/flutter_tools/test/general.shard/commands/build_appbundle_test.dart

```diff
@@ -122,6 +123,64 @@ void main() {
       tryToDelete(tempDir);
     });
 
+    group('AndroidSdk', () {
+      FileSystem memoryFileSystem;
+
+      setUp(() {
+        memoryFileSystem = MemoryFileSystem();
+
+        tempDir = memoryFileSystem.systemTempDirectory.createTempSync('flutter_tools_packages_test.');
+        memoryFileSystem.currentDirectory = tempDir;
+
+        gradlew = memoryFileSystem.path.join(tempDir.path, 'flutter_project', 'android',
+            platform.isWindows ? 'gradlew.bat' : 'gradlew');
+      });
+
+      testUsingContext('validateSdkWellFormed() not called, sdk reinitialized', () async {
+        final Directory gradleCacheDir = memoryFileSystem.directory('/flutter_root/bin/cache/artifacts/gradle_wrapper')..createSync(recursive: true);
+        gradleCacheDir.childFile('gradlew').createSync();
```

Comment:
`gradleCacheDir.childFile('gradlew')` -> `gradleCacheDir.childFile(gradlew)`.
I think the test on Windows is failing because of this issue.

Reply:
same comment for the rest of the test files.

Reply:
Ahh thanks.  I can't use the other variable because this is actually the one from the flutter cache bin right?